website_sale_qty/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'Website Sale - Price Tiers',
+    'summary': 'Add price tier radio buttons to product pages in website shop',
+    'version': '9.0.1.0.0',
+    'category': 'E-commerce',
+    'website': 'https://laslabs.com/',
+    'author': 'LasLabs',
```

Comment:
This one is correct. It should include "Odoo Community Association (OCA)" for indexing purposes.